src/main/java/com/starkinfra/IndividualDocument.java

```diff
@@ -0,0 +1,660 @@
+     * <p>
+     * Return:
+     * @return IndividualDocument.Page object:
+     * IndividualDocument.Page.documents: list of IndividualDocument objects with updated attributes
+     * IndividualDocument.Page.cursor: cursor to retrieve the next page of IndividualDocument objects
+     * @throws Exception error in the request
+     */
+    public static Page page() throws Exception {
+        return page(new HashMap<>(), null);
+    }
+
+    public final static class Log extends Resource {
+        static ClassData data = new ClassData(Log.class, "IndividualDocumentLog");
+
+        public String created;
+        public String type;
+        public List<ErrorElement> errors;
+        public IndividualDocument document;
+
+        /**
+         * IndividualDocument Log object
+         * <p>
+         * Every time a IndividualDocument entity is modified, a corresponding IndividualDocument Log
+         * is generated for the entity. This log is never generated by the user.
+         * <p>
+         * Parameters:
+         * @param id [string]: unique id returned when the log is created. ex: "5656565656565656"
+         * @param document [IndividualDocument]: IndividualDocument entity to which the log refers to.
+         * @param errors [list of strings]: list of errors linked to the IndividualDocument event.
+         * @param type [string]: type of the IndividualDocument event which triggered the log creation. ex: "processing" or "success"
+         * @param created [string]: creation datetime for the log. ex: "2020-03-10"
+         */
+
+        public Log(String created, String type, List<ErrorElement> errors, IndividualDocument document, String id) {
+            super(id);
+            this.created = created;
+            this.type = type;
+            this.errors = errors;
+            this.document = document;
+        }
+
+        /**
+         * Retrieve a specific IndividualDocument Log
+         * <p>
+         * Receive a single IndividualDocument Log object previously created by the Stark Infra API by passing its id
+         * <p>
+         * Parameters:
+         * @param id [string]: IndividualDocument unique id. ex: "5656565656565656"
+         * <p>
+         * Return:
+         * @return IndividualDocument.Log object with updated attributes
+         * @throws Exception error in the request
+         */
+        public static Log get(String id) throws Exception {
+            return Log.get(id, null);
+        }
+
+        /**
+         * Retrieve a specific IndividualDocument Log
+         * <p>
+         * Receive a single IndividualDocument Log object previously created by the Stark Infra API by passing its id
+         * <p>
+         * Parameters:
+         * @param id [string]: IndividualDocument unique id. ex: "5656565656565656"
+         * @param user [Project object]: Project object. Not necessary if StarkInfra.Settings.user was set before function call
```

Comment:
@param user [Organization/Project object]: Organization or Project object. Not necessary if starkinfra.User.defaultUser was set before function call

Bom dar um search-all caso tenha em outro lugar isso